### PR TITLE
refactor: Fix code smells

### PR DESCRIPTION
**Pull request by Judge40**
TIS21-1397

### CODE REVIEW

File: src/main/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResource.java [hunk 1]
```diff
@@ -155,7 +155,6 @@ public FormRPartAResource(FormRPartAService service, FormRPartAValidator validat
   @GetMapping("/formr-parta/{id}")
   public ResponseEntity<FormRPartADto> getFormRPartAsById(@PathVariable String id,
       @RequestHeader(HttpHeaders.AUTHORIZATION) String token) {
-    // TODO: verify the formR PartA for the id belongs to the trainee
```
Comment (ReubenRobertsHEE) [inline src/main/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResource.java:158]: Is this TODO resolved somewhere or no longer relevant? I vaguely recall some discussions around securing access to the saved forms, but I'm not sure if that is related.

Comment (ReubenRobertsHEE) [reply src/main/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResource.java:158]: Otherwise it all looks fine to me.

Comment (Judge40) [reply src/main/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResource.java:158]: The service takes the trainee ID as a parameter (retrieved from the token) which is then used to build the path to the form. If the trainee ID isn't correct for the form ID then it won't exist on the built path.

As far as I could see, the concerns raised in the TODO are covered already.

Comment (Judge40) [reply src/main/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResource.java:158]: The service takes the trainee ID as a parameter (retrieved from the token) which is then used to build the path to the form. If the trainee ID isn't correct for the form ID then it won't exist on the built path.

As far as I could see, the concerns raised in the TODO are covered already.

**Review by ReubenRobertsHEE — APPROVED** [review]
LGTM

**Pull request merged**